ui/oopsyraidsy/data/04-sb/trial/suzaku-ex.ts

```diff
@@ -0,0 +1,78 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+import { playerDamageFields } from '../../../oopsy_common';
+
+// TODO: Rekindle (32E0) hitting multiple adds, any other players,
+// or hitting adds before they've been killed once is a mistake.
+// However, there is a point where it should hit one add in the
+// beginning. This also happens later in the fight too, where it
+// should not hit anybody else.
+
+// TODO: what is getting hit by an orb during Close-Quarter Crescendo?
+
+export type Data = OopsyData;
+
+// Suzaku Extreme
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.HellsKierExtreme,
+  damageWarn: {
+    'SuzakuEx Rout': '32F0', // untargetable phase 1 charge across the arena
+    'SuzakuEx Fleeting Summer': '32D3', // targeted conal
+    'SuzakuEx Scarlet Tail Feather Wing And A Prayer': '32D4', // circle aoe from unkilled plume
+    'SuzakuEx Scarlet Plume Wing And A Prayer': '3244', // circle aoe from unkilled plume
+    'SuzakuEx Ashes To Ashes': '32D0', // Scarlet Lady add, raidwide explosion if not killed in time
+    'SuzakuEx Well Of Flame': '32E1', // targeted wide line aoe
+    'SuzakuEx Hotspot': '32E2', // platform fire when the runes are activated
+  },
+  damageFail: {
+    'SuzakuEx Immolate': '32E6', // Tower mechanic failure on Incadescent Interlude (party failure, not personal)
+    'SuzakuEx Phantom Flurry': '32DE', // "phantom half" final hit of Phantom Flurry
+  },
+  soloWarn: {
+    'SuzakuEx Scathing Net': '3243', // stack marker
```

Comment:
This one *might* be worth handling as a trigger, since it can be handled with Diamondback on Blue Mage.

Reply:
Should I just remove it, then?

Reply:
Oh, right, that's why you left it as a warning and not a failure? It's probably fine then. It just happened to be fresh in my head since I ran it with some buddies on Wednesday.